#!/bin/sh /usr/share/dpatch/dpatch-run
## 04-shebang.dpatch by Daniel Baumann <daniel@debian.org>
##
## DP: Correcting shebang.

@DPATCH@

diff -Naurp openerp-server.orig/bin/addons/auction/barcode/test.py openerp-server/bin/addons/auction/barcode/test.py
--- openerp-server.orig/bin/addons/auction/barcode/test.py	2008-12-16 11:06:21.000000000 +0000
+++ openerp-server/bin/addons/auction/barcode/test.py	2008-12-16 11:27:19.000000000 +0000
@@ -1,4 +1,4 @@
-#!/usr/pkg/bin/python
+#!/usr/bin/python
 from common import *
 from code39 import *
 from code93 import *
diff -Naurp openerp-server.orig/bin/addons/document/ftpserver/ftpserver.py openerp-server/bin/addons/document/ftpserver/ftpserver.py
--- openerp-server.orig/bin/addons/document/ftpserver/ftpserver.py	2008-12-16 11:06:22.000000000 +0000
+++ openerp-server/bin/addons/document/ftpserver/ftpserver.py	2008-12-16 11:28:19.000000000 +0000
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/python
 # ftpserver.py
 #
 #  pyftpdlib is released under the MIT license, reproduced below:
